common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -161,3 +161,35 @@ def test_demand_hint(self):
                 {'event': {u'hint_index': 0, u'hint_len': 2, u'hint_text': u'demand-hint1'}}
             ],
             actual_events)
+
+
+class ProblemPartialCredit(ProblemsTest):
+    """
+    Makes sure that the partial credit is appearing properly.
+    """
+    def get_problem(self):
```

Comment:
The base class (ProblemsTest, at the top of this file) is the one that calls self.get_problem() (line 36 at the moment). If you check out ProblemExtendedHintTest and ProblemClarificationTest, they also don't explicitly call their get_problem() methods.

Does that help, or were you asking about something else?